.storybook/config.js

```diff
@@ -1,22 +1,20 @@
+import isChromatic from 'storybook-chromatic/isChromatic';
+
 import fetchMock from 'fetch-mock';
-import {
-    configure,
-    addParameters
-} from '@storybook/react';
+import { configure, addParameters } from '@storybook/react';
 
-import {
-    sharecount
-} from '@root/fixtures/article';
-import {
-    commentCount
-} from '@root/fixtures/commentCounts';
+import { sharecount } from '@root/fixtures/article';
+import { commentCount } from '@root/fixtures/commentCounts';
 
-import {
-    defaults
-} from './default-css';
+import { defaults } from './default-css';
 
 import 'reset-css';
 
+import { Lazy } from '@root/src/web/components/Lazy';
+
+// Prevent components being lazy rendered when we're taking Chromatic snapshots
+Lazy.disabled = isChromatic();
```

Comment:
I assume this is only temporary? it would be great to render lazy loaded things in the future

Reply:
We still render them, just not in a lazy manner